GitUI/UserControls/RevisionGrid/RevisionGridControl.cs

```diff
@@ -2493,6 +2493,23 @@ private void compareSelectedCommitsMenuItem_Click(object sender, EventArgs e)
             UICommands.ShowFormDiff(IsFirstParentValid(), baseCommit.ObjectId, headCommit.ObjectId, baseCommit.Subject, headCommit.Subject);
         }
 
+        private void diffSelectedCommitsMenuItem_Click(object sender, EventArgs e)
```

Comment:
As you already have prepared, a menu item should be added, too.

Reply:
Added

Reply:
Thank you. Could you display the configured Hotkey in the `openCommitsWithDiffToolMenuItem`, please, and set a mnemonic, e.g. "Open selected commits with &difftool".

Reply:
Added the mnemonic (only change in addition to the rebase).
HotKey already displayed